sklearn/decomposition/dict_learning.py

```diff
@@ -1036,18 +1108,23 @@ class MiniBatchDictionaryLearning(BaseEstimator, SparseCodingMixin):
     alpha : float,
         sparsity controlling parameter
 
+    l1_ratio: float,
+        sparsity controlling parameter, for 'elastic_net' method only
+
     n_iter : int,
         total number of iterations to perform
 
-    fit_algorithm : {'lars', 'cd'}
+    fit_algorithm : {'lars', 'cd', 'elastic_net'}
```

Comment:
Also does is the current implementation well defined and equivalent to l1-ball projection when `l1_ratio=1.0`?